app/assets/javascripts/core/comment_controller.coffee

```diff
@@ -35,6 +39,14 @@ class Spaces.CommentsController extends Spaces.ViewController
 			error: ->
 				console.log "Error getting all names"
 
-
-
-		
+	addUploadPreview: =>
+		($ "form#new_comment input[type='file']").attr "onchange", "readURL(this)"
+		window.readURL = @readURL
```

Comment:
If I don't make readURL accessible globally then it's not in the scope of the `input`'s `onchange` attribute